src/selectors/columnWidths.js

```diff
@@ -78,19 +78,14 @@ function columnWidths(columnGroupProps, columnProps, scrollEnabledY, width) {
  */
 function flexWidths(columnGroupProps, columnProps, viewportWidth) {
   let remainingFlexGrow = getTotalFlexGrow(columnProps);
-  if (remainingFlexGrow === 0) {
```

Comment:
@Invizor, the only purpose of lines [88](https://github.com/schrodinger/fixed-data-table-2/blob/v1.0-beta/src/selectors/columnWidths.js#L88) to [105](https://github.com/schrodinger/fixed-data-table-2/blob/v1.0-beta/src/selectors/columnWidths.js#L105) is to [calculate the new column widths](https://github.com/schrodinger/fixed-data-table-2/blob/v1.0-beta/src/selectors/columnWidths.js#L103) IF flex is present.
So if flex is not present (i.e. `remainingFlexGrow === 0`), then we can just ignore the section of code.

How about we keep the check for `remainingFlexGrow` and only calculate `newColumnProps` IF `remainingFlexGrow` is not zero? Otherwise we'll use the existing columnProps.

Something like
```js
  let remainingFlexGrow = getTotalFlexGrow(columnProps);
  let newColumnProps = columnProps;

  // if any column is a flex column
  if (remainingFlexGrow !== 0) {
    /* ... calculate newColumnProps ... (line 88 to 105) */
  }
```